tests/qasm_test.py

```diff
@@ -57,6 +57,45 @@ def test_pytket_qir_qasm_ii() -> None:
         check_qir_result(result, "test_pytket_qir_qasm")
 
 
+def test_pytket_qir_qasm_long() -> None:
+    try:
```

Comment:
Not sure if you intended to include this test, which will never run on CI? Better to check in the circuit files with a separate test script that can be run locally; otherwise this is useless unless one happens to have the files.

Reply:
I am running this test locally, if you think this should be in a separate file and not in the repo I am happy to move it.
(There is a similar case above)

Reply:
(The `qec_ii.qasm` file is also not in the repo)

Reply:
See below